psi4/src/CMakeLists.txt

```diff
@@ -26,10 +26,9 @@ add_library(core SHARED ${sources_list})
 
 get_property(psi4_binmodules GLOBAL PROPERTY BINLIST)
 if(ENABLE_psi4pasture)
-  target_link_libraries(core PRIVATE ${PRE_LIBRARY_OPTION} ${psi4_binmodules} ${POST_LIBRARY_OPTION})
-else()
-  target_link_libraries(core PRIVATE ${psi4_binmodules})
+  target_link_libraries(core PRIVATE ${PRE_LIBRARY_OPTION} qt ${POST_LIBRARY_OPTION})
```

Comment:
this will be fine for now. I wrote up last week the problems with _LIBRARY_OPTION, but that comment seems to have dematerialized. Basically, in doing this you're adding linker directives (`-Wl,--stuff`) as libraries, and cmake is known to rearrange them according to library rules. So `-llib1 -llib2 -llib3 -llib2` is perfectly fine optimized to `-llib1 -llib3 -llib2`. But when `lib2` isn't a library but `-Wl,--whole-archive` or `-Wl,--start-group`, then that "optimization" is a corruption. 

So adding this is just fine for now, but whenever we get more than one set of archiving directives (there's another one lurking for PCMSolver), then we'll have trouble. Targets get around this, which is partly why I'm gung-ho on them.